docs/docs/graphql.md

```diff
@@ -1506,16 +1513,34 @@ The following is some basic setup information, but please consult [Docusaurus](h
 2. Create a `docs` directory at the root of your project
 
 ```terminal
-mcd docs
-`
+mkdir docs
```

Comment:
After running `npx create-docusaurus@latest docs classic` - the docs folder was already created and populated. I ran this in my root test project folder.

Reply:
@pantheredeye Do you think this instruction should change to "cd docs"?

Reply:
```suggestion
cd docs
```

Reply:
Changing to 2. Ensure a `docs` directory exists at the root of your project